### DISCUSSION TITLE
Add property IsVisible to Section

### DISCUSSION THREAD

**Discussion by ghost** [Live-Charts/LiveCharts2#172] [Ideas]
Add property **IsVisible** to class **RectangularSection** (similar to **Series** and **Axis**) to switch visibility of the RectangularSection on and off.

**Comment by beto-rodriguez** [comment]
This was definitely missing.

It was added with 2c146aa906ddad46c3a08b5f22b2bed03f102910 and will be included in the next release

**Reply by ghost** [reply]
Thank you! Works perfectly.

**Reply by MithrilMan** [reply]
can we have an IsAnimated property too to disable animations on specific sections?
Using them as background to logical split chart areas doesn't look good with animations
